docs/DeveloperGuide.adoc

```diff
@@ -297,33 +297,69 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |secretary of NUS CCA  |manually add new interviewees and their availabilities| make changes without having to edit the .csv file and doing another import.
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |secretary of NUS CCA  |delete an interview |remove interviews that have been taken out.
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |secretary of NUS CCA  |find a interviewee by name|locate details of person and their interview without having to go through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |secretary of NUS CCA |automate the process of scheduling interviews |lighten my workload
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* * *` |secretary of NUS CCA |import the available timeslots of interviewees from a .csv template |schedule the interviews for them
+
+|`* * *` |secretary of NUS CCA |export the scheduled interviews as a .csv file |I can view and manage them in Excel
+
+|`* * *` |secretary of NUS CCA |add details of interviewee e.g. email/phone number |easily view the details of the interviewees
+
+|`* * *` |secretary of NUS CCA |view a timetable of the finalized interview slots |show the interviewers which timeslots they will be taking at a glance
+
+|`* * *` |secretary of NUS CCA |email blast the generated interview schedule to all interviewees |inform the interviewees of their interview
+
+|`* * *` |secretary of NUS CCA |be informed if interview allocation of a certain interviewee has failed |manually fix conflicts or get the interviewee to give another available timeslot
+
+|`* * *` |secretary of NUS CCA |automate the process of scheduling interviews |lighten my workload
+
+|`* * *` |secretary of NUS CCA |mark an interviewee as 'completed interview' |track and manage uncompleted interviews
+
+|`* * *` |secretary of NUS CCA |add multiple tags to an interviewee in a single command |I can be efficient
+
+|`* * *` |fast-typer |use CLI for the app instead of mouse/GUI |access the commands in the fastest possible way
+
+|`* *` |developer |CLI and GUI to be separated |isolate either one for testing and debugging purposes
```

Comment:
Should the developer of the application be considered a user? I think user stories should pertain to users only.